crowdin.yml

```diff
@@ -0,0 +1,22 @@
+# Refer to https://developer.crowdin.com/configuration-file/.
+
+"preserve_hierarchy": true
+
+files:
+  [
+    {
+      "source": "docs/**/*",
+      "translation": "translations/%two_letters_code%/%original_path%/%original_file_name%",
+      "ignore":
+        [
+          "docs/api/qiskit/**/*",
+          "docs/api/qiskit-ibm-runtime/**/*",
+          "docs/api/qiskit-ibm-provider/**/*",
```

Comment:
wouldn't it simply be easy to have

```
"/docs/api",
```

instead?

Reply:
No because of `docs/api/migration-guides`.

Reply:
Do you want the migration guides to be translated?

Reply:
I'm not sure - what do you think? See https://docs.quantum.ibm.com/api/migration-guides.

Reply:
I'll ask the team 👍

Reply:
Ok, let's keep it then :D